Modules/Sources/DesignSystem/Foundation/Font+DesignSystem.swift

```diff
@@ -23,3 +23,60 @@ public extension Font {
         public static let caption = Font.caption
     }
 }
+
+public extension Font.DS {
+    static func font(_ style: DesignSystem.TextStyle) -> Font {
+        switch style {
+        case .heading1:
+            return Font.DS.heading1
+
+        case .heading2:
+            return Font.DS.heading2
+
+        case .heading3:
+            return Font.DS.heading3
+
+        case .heading4:
+            return Font.DS.heading4
+
+        case .bodySmall(let weight):
+            switch weight {
+            case .regular:
+                return Font.DS.Body.small
+            case .emphasized:
+                return Font.DS.Body.Emphasized.small
+            }
+
+        case .bodyMedium(let weight):
+            switch weight {
+            case .regular:
+                return Font.DS.Body.medium
+            case .emphasized:
+                return Font.DS.Body.Emphasized.medium
+            }
+
+        case .bodyLarge(let weight):
+            switch weight {
+            case .regular:
+                return Font.DS.Body.large
+            case .emphasized:
+                return Font.DS.Body.Emphasized.large
+            }
+
+        case .footnote:
+            return Font.DS.footnote
+
+        case .caption:
+            return Font.DS.caption
+        }
+    }
+
+//    var `case`: Text.Case? {
+//        switch self {
+//        case .caption:
+//            return .uppercase
+//        default:
+//            return nil
+//        }
+//    }
```

Comment:
I suppose this needs to be deleted as it is moved under `TextStyle`

Reply:
Yes, I thought I did. I'll do it again.